setup.py

```diff
@@ -114,5 +88,14 @@ def resolve_install_requires():
     },
     extras_require={
         'fast': ['pycohttpparser'],
+        # Fallback to good SSL on bad Python versions.
+        ':python_full_version < "2.7.9" or python_version == "3.3"': [
```

Comment:
You don't indicate that 3.3 is supported in the classifiers. Why are you accommodating it here then?